docker-compose.yml

```diff
@@ -120,6 +120,9 @@ services:
       F8A_UNCLOUDED_MODE: 'true'
       AWS_S3_ACCESS_KEY_ID: GNV3SAHAHA3DOT99GQII
       AWS_S3_SECRET_ACCESS_KEY: ZmvMwngonaDK5ymlCd6ptaalDdJsCn3aSSxASPaZ
+      AWS_S3_IS_LOCAL: 1
+      AWS_EPV_BUCKET: "${USER}-bayesian-core-data"
+      AWS_PKG_BUCKET: "${USER}-bayesian-core-package-data"
```

Comment:
This would introduce a lot of complexity on worker side with duplicit logic. Please use already existing adapters present in f8a_worker/storages/s3_*.py depending on what you want to use.

You can easily do:

```python
from selinon import StoragePool

# obtain S3 adapter for package-version level data:
s3 = StoragePool.get_connected_storage('S3Data')

task_result = s3.retrieve_task_result(ecosystem, package_name, version, task_name)

# Similarly to obtain S3 adapter for package level data:
s3 = StoragePool.get_connected_storage('S3PackageData')

task_result = s3.retrieve_task_result(ecosystem, package_name, task_name)
```

Also `AWS_S3_IS_LOCAL` is not needed. Whether we run locally or in the cloud should be transparent - we should share code logic otherwise deploying to production would get different behaviour then your local setup.